.github/workflows/push.yml

```
@@ -1,167 +1,49 @@
-      env:
-        API_TOKEN_GITHUB: ${{ secrets.GH_TOKEN }}    
-      with:
-        source_file: 'talawa-api-docs/'
-        destination_repo: 'PalisadoesFoundation/talawa-docs'
-        destination_branch: 'testing-workflows'
-        destination_folder: 'docs/'
-        user_email: '${{env.email}}'
-        user_name: '${{github.actor}}'
-        commit_message: 'Talawa API docs updated'
-      
+name: Push Workflow for develop
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)